h/static/styles/partials-v2/_search.scss

```diff
@@ -225,15 +225,27 @@
                  // work).
 }
 
+$stats-icon-column-width: 20px;
```

Comment:
Please do :) Making the rationale for particular sizes in CSS clear makes it easier to avoid breaking things when refactoring.